tests/test_monotonic_constraints.py

```
@@ -0,0 +1,1287 @@
+        }
+        
+        params = {
+            "control_variates": False,
+            "split_score_func": "L2",
+            "monotonic_constraints": monotonic_constraints
+        }
+        
+        optimizer = {'algo': 'SGD', 'lr': 0.5, 'start_idx': 0, 'stop_idx': 1}
+        
+        with self.assertRaises(ValueError) as context:
+            model = GBTModel(
```

Ruff (0.14.14)

[error] Local variable `model` is assigned to but never used

Remove assignment to unused variable `model`

(F841)